### DISCUSSION TITLE
How to implement a custom filter for OR logic

### DISCUSSION THREAD

**Discussion by Dazfl** [Biarity/Sieve#183] [General]
It appears (for now) that the Sieve filters doesn't handle a scenario like `filters=Property1@=value1|Property2@=value2`, so I figured a custom filter is the way to go.  But I'm a bit stuck as to how to implement this.

Does anyone know how you could implement a custom filter to implement OR logic that equates to something like:

`Property1 == 'value1' OR Property2 == 'value2' ...`

... how would you use it?  Would it look like - `filters=CustomFilter@=(value1, value2, ...)`??

**Comment by asafblum** [comment]
Hi @Dazfl Any luck here? what was you solution for IN List?

**Reply by Dazfl** [reply]
Hi @asafblum ... I ended up writing a custom filter (it was a while ago now), but sadly I've stopped using this library because it hasn't been worked on in quite a while.  I've now started using OData (although I'm still evaluating it).
